dot-studio/svc_coord

```diff
@@ -52,13 +52,13 @@ function wait_for_svc_to_load() {
 
 document "wait_for_port_to_listen" <<DOC
   Wait for a port to be listening.
-
   @(arg:1) Port to wait for to be listening
   @(arg:2) Wheter or not this process runs silently
 DOC
 function wait_for_port_to_listen() {
   install_if_missing core/busybox-static netstat
-  
+  echo " => deprecated use wait_or_fail_for_port_to_listen"
```

Comment:
If this is the case, then don't perform any action at all. Just post the message (e.g. https://github.com/chef/ci-studio-common/blob/master/dot-studio/hab_toolchain#L42-L45)

Reply:
@tduffield We do not want to break the current projects that are using this function. I was thinking we would update the projects we know are using this function and then change to not performing any action like your example.

Reply:
@lancewf I'm down with that as long as there is a forcing factor to update. Either a time/date that we will remove it, or a follow up task to manually go through all the projects.

Reply:
@tduffield That sounds good. I created a task here https://github.com/chef/ingest-service/issues/139 to update projects to use the new function and then remove it from this project once done.